src/main.js

```diff
@@ -48,7 +48,9 @@ mainEvent.on("restart", () => {
 
 async function createWindow() {
   log.info(
-    "Welcome to the UBports Installer version " + packageInfo.version + "!"
+    "Welcome to the UBports Installer version " +
+      global.packageInfo.version +
+      "!"
```

Comment:
```suggestion
    "Welcome to the UBports Installer version " + packageInfo.version + "!"
```

this should not have been changed back and probably happened during conflict resolution. we tried to get rid of the globals a while back wherever possible (not everywhere yet, but here it works).